packages/rich-text-editor/src/RichTextEditor/keymap.ts

```diff
@@ -70,8 +78,22 @@ export function buildKeymap(schema: Schema) {
   }
 
   if (schema.nodes.paragraph) {
-    const type = schema.nodes.paragraph
-    keys["Shift-Ctrl-0"] = setBlockType(type)
+    keys["Shift-Ctrl-0"] = setBlockType(schema.nodes.paragraph)
+  }
+
+  if (schema.nodes.bullet_list) {
+    keys["Shift-Ctrl-8"] = wrapInList(schema.nodes.bullet_list)
+  }
+
+  if (schema.nodes.ordered_list) {
+    keys["Shift-Ctrl-9"] = wrapInList(schema.nodes.ordered_list)
+  }
+
+  if (schema.nodes.list_item) {
+    const type = schema.nodes.list_item
+    keys["Enter"] = splitListItem(type)
+    keys["Mod-["] = liftListItem(type)
+    keys["Mod-]"] = sinkListItem(type)
   }
```

Comment:
Nice! Definitely think it worth investing time after the toolbar has been integrate to pass these commands into the tooltips